### PR TITLE
Add `Astro.callAction()` guide

### LINKED ISSUE #9458: Missing docs for `Astro.callAction` in `guides/actions`
### 📚 Subject area/topic

Actions

### 📋 Suggested page

https://docs.astro.build/en/guides/actions/

### 📋 General description or bullet points (if proposing new content)

There are completely missing docs (on this page) for correct calls from server-side Astro component.
If we call it from Astro component with `actions.getGreeting ` instead of `Astro.callAction(...)`, the context is non-existing.
I was just scratching my head for a few minutes until I found out the correct way.

```ts
// ❌ Wrong
await actions.getGreeting({ name: "Bob" });
// ✅ Correct
await Astro.callAction(actions.getGreeting, { name: "Bob" });
 ```

### 🖥️ Reproduction of code samples in StackBlitz

_No response_

### PR DESCRIPTION
**Pull request by bholmesdev**
#### Description (required)

Add guide section to call actions from server code.

#### Related issues & labels (optional)

- Closes #9458 
- Suggested label: add new content

### CODE REVIEW

File: src/content/docs/en/guides/actions.mdx [hunk 1]
```diff
@@ -581,3 +581,21 @@ export const server = {
 }
 ```
 :::
+
+## Call actions from Astro components and server endpoints
+
+You may want to call actions directly from Astro component scripts or server endpoints. This is common when reusing logic from your actions in other server code.
+
+Call actions using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](http://localhost:4321/en/guides/endpoints/#server-endpoints-api-routes)), passing the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:
```
Comment (casungo) [inline src/content/docs/en/guides/actions.mdx:589]: It's quite optimistic to assume that every user always has a development server of the documentation up and running. ;)

```suggestion
Call actions using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](/en/guides/endpoints/#server-endpoints-api-routes)), passing the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:
```

**Review by casungo — CHANGES_REQUESTED** [review]
LGTM! just fixed a link

File: src/content/docs/en/guides/actions.mdx [hunk 2]
```diff
@@ -581,3 +581,21 @@ export const server = {
 }
 ```
 :::
+
+## Call actions from Astro components and server endpoints
+
+You may want to call actions directly from Astro component scripts or server endpoints. This is common when reusing logic from your actions in other server code.
+
+Call actions using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](/en/guides/endpoints/#server-endpoints-api-routes)), passing the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:
```
Comment (sarah11918) [inline src/content/docs/en/guides/actions.mdx:589]:
```suggestion
You can call actions directly from Astro component scripts using the `Astro.callAction()` wrapper (or `context.callAction()` when using a [server endpoint](/en/guides/endpoints/#server-endpoints-api-routes)). This is common to reuse logic from your actions in other server code. 

Pass the action as the first argument and any input parameters as the second argument. This returns the same `data` and `error` objects you receive when calling actions on the client:
```
Just removing the whole "you might want to" in favour of just saying how to do it.

**Review by sarah11918 — COMMENTED** [review]
Looks good @bholmesdev ! Just tightened up the intro, and then as I think this is missing documentation (not waiting on a feature), this can merge as soon as you're happy with it.

**Pull request merged**